uninstall.sh

```diff
@@ -5,20 +5,36 @@ set -euo pipefail
 DIR_NAME="$(dirname $0)"
 
 source "${DIR_NAME}/utils/common.sh"
-
-KAS_INSTALLER_ENV_FILE="kas-installer.env"
-source ${KAS_INSTALLER_ENV_FILE}
+source "${DIR_NAME}/kas-installer.env"
+source "${DIR_NAME}/kas-installer-defaults.env"
 
 if ! cluster_domain_check "${K8S_CLUSTER_DOMAIN}" "uninstall"; then
     echo "Exiting ${0}"
     exit 1
 fi
 
 KAS_FLEET_MANAGER_DIR="${DIR_NAME}/kas-fleet-manager"
-KAS_FLEET_MANAGER_DEPLOY_ENV_FILE="${KAS_FLEET_MANAGER_DIR}/kas-fleet-manager-deploy.env"
-source ${KAS_FLEET_MANAGER_DEPLOY_ENV_FILE}
+source "${KAS_FLEET_MANAGER_DIR}/kas-fleet-manager-deploy.env"
+
+if [ -z "$(${OC} get namespace/${KAS_FLEET_MANAGER_NAMESPACE} -o jsonpath="{.metadata.name}" --ignore-not-found)" ]; then
+    echo "namespace/${KAS_FLEET_MANAGER_NAMESPACE} is already removed"
+else
+    # Remove all Kafka instances
+    for MKID in $(${DIR_NAME}/managed_kafka.sh --list | jq -r '.items[] | .id' 2>/dev/null || echo "") ; do
```

Comment:
same comment as https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/pull/272#discussion_r1105626101

Reply:
Here we want to remove all instances regardless of the cluster. The pagination is built-in to `managed_kafka.sh`

Reply:
The case for this is any remaining statically-configured clusters with Kafkas deployed.

Reply:
You are right. I initially thought this is only for dedicated clusters.

ignore this comment.